tests/builders/test_response_builder.py

```diff
@@ -44,7 +47,9 @@ def _get_builder_mock(expected_value: Any) -> Union[ContentBuilder, HeaderBuilde
                 "application/json": {
                     "schema": {
                         "type": "string",
-                    }
+                    },
+                    "example": "an example",
+                    "examples": ["example1", "example2"]
```

Comment:
I would assume since we have two items in this `examples` list it should be reflected in the expected value as well, am I missing something?